src/Components/WebAssembly/testassets/Wasm.Authentication.Client/Shared/RedirectToLogin.razor

```diff
@@ -1,10 +1,12 @@
 ﻿@inject NavigationManager Navigation
+@using Microsoft.AspNetCore.Components.WebAssembly.Authentication
 @using Microsoft.Extensions.Options
+
 @inject IOptionsSnapshot<RemoteAuthenticationOptions<ApiAuthorizationProviderOptions>> Options
 @code {
 
     protected override void OnInitialized()
     {
-        Navigation.NavigateTo($"{Options.Get(Microsoft.Extensions.Options.Options.DefaultName).AuthenticationPaths.LogInPath}?returnUrl={Uri.EscapeDataString(Navigation.Uri)}");
+        Navigation.NavigateToLogin(Options.Get(Microsoft.Extensions.Options.Options.DefaultName).AuthenticationPaths.LogInPath);
```

Comment:
Is this the expected usage pattern for customers? It's very verbose. Isn't it practical for `NavigateToLogin` to get the path from these options by default?

Reply:
This is only for our tests. On the templates we do `Navigation.NavigateToLogin("authentication/login")` directly, as it is clearer for customers what is going on.

Reply:
For additional context, this is how we do it on the template right now:
https://github.com/dotnet/aspnetcore/blob/93d741edb0fa195f59d13743c35d21ee59099fc1/src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Client/Shared/LoginDisplay.IndividualLocalAuth.razor#L32